src/mito2/src/manifest/action.rs

```diff
@@ -114,16 +115,22 @@ impl RegionManifestDataBuilder {
         }
     }
 
-    pub fn try_build(self) -> Result<RegionManifestData> {
-        Ok(RegionManifestData {
+    pub fn try_build(self) -> Result<RegionManifest> {
+        Ok(RegionManifest {
             metadata: self.metadata.context(RegionMetadataNotFoundSnafu)?,
-            version: self.version,
-            committed_sequence: self.committed_sequence,
+            version: self.version.context(RegionVersionNotFoundSnafu)?,
         })
     }
 }
 
-// The checkpoint of region manifest, generated by checkpoint.
+/// The region version checkpoint
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct RegionVersion {
```

Comment:
Could we rename it to `RegionVersionCheckpoint`, `VersionCheckpoint`, or `ManifestCheckpoint`? `RegionVersion` is easy to confuse with the existing `struct Version` under the `region` mod.

Reply:
I lost in those version/sequence/number. Need a mermaid to cure :sob: